pittapi/sports.py

```diff
@@ -129,6 +130,7 @@ def get_next_football_game() -> dict:
             "status" : status
         }
     except IndexError:
+        # IndexError occurs when a next game on the schedule is not present, implying it's the offseason.
```

Comment:
Let' just return `status: NO_GAME_SCHEDULED` or something like it. There may be other reasons no future game is scheduled -- strikes, NCAA imposed penalties, etc. Instead if attempting to interpret all possible causes, let's just be very exact and say exactly why an `IndexError` may occur, i.e., there is no next game.